### PR TITLE
Henk's review feedback for WGLC

### PR DESCRIPTION
**Pull request by henkbirkholz**
a little bit late, but there are always little fires everywhere.

### CODE REVIEW

**Review by dthaler — COMMENTED** [review]
This looks ok to me, please rebase and mark as ready for review

File: draft-ietf-rats-endorsements.md [hunk 1]
```diff
@@ -62,73 +62,73 @@ message format for Endorsements in the scope of the RATS architecture.
...
 Appraisal policies (Appraisal Policy for Evidence, and Appraisal Policy for
-Attestation Results) involve comparing the actual state of an Attester against
+Attestation Results) involve comparing the actual-state of an Attester against
 desired or undesired states, in order to determine how trustworthy the Attester
 is for its purposes.  The state of an Attester represents its composition of
-components of execution environments (its "shape"), typically in a hierarchical
-manner.  The state of an Attester also encompasses the combination of static and
-dynamic constitution (e.g., provisioned and deployed software, firmware, and
+components of execution environments (its "shape" or "composition"), typically in a hierarchical
+manner, i.e., a tree graph.  The state of an Attester also encompasses the combination of static and
```
Comment (thomas-fossati) [inline draft-ietf-rats-endorsements.md]: I have never seen "tree graph".  You could say "tree" or "connected and acyclic graph".  My preference is for the former.

File: draft-ietf-rats-endorsements.md [hunk 2]
```diff
@@ -62,73 +62,73 @@ message format for Endorsements in the scope of the RATS architecture.
...
-messages with information about actual state, and information about desired/undesired
+conceptual messages with information about actual state, and information about desired/undesired
 states, and an appraisal policy that controls how the two are compared.
 
 Each Attester in general has at least one Attesting Environment and one Target
 Environment (e.g., hardware, firmware, Operating System, etc.).  Typically, each
 Attester has multiple Target Environments, each with their own set of claims (sometimes
-called a "claimset") representing their actual state.  Additionally, the number of
-Target Environments is not limited.
+called a "claim sets") representing their actual-state.  Additionally, the number of
```
Comment (thomas-fossati) [inline draft-ietf-rats-endorsements.md]: I prefer the non-hyphenised version "actual state".  Besides, the term should be made consistent across the whole document.

File: draft-ietf-rats-endorsements.md [hunk 3]
```diff
@@ -62,73 +62,73 @@ message format for Endorsements in the scope of the RATS architecture.
...
+Target Environments and Attesting Environments that are components of an Attester are not limited.
```
Comment (thomas-fossati) [inline draft-ietf-rats-endorsements.md]: I suggest removing the sentence _"[A]dditionally, [...] not limited"_ as it doesn't add any value to the paragraph.

File: draft-ietf-rats-endorsements.md [hunk 4]
```diff
@@ -62,73 +62,73 @@ message format for Endorsements in the scope of the RATS architecture.
...
 
-"Actual state" is a group of claimsets about the actual state of the Attester at a
-given point in time. Each claimset holds claims about a specific Target Environment
+"Actual-state" is a group of claim sets about the actual state of the Attester at a
+given point in time. Each claim set holds claims about a specific Target Environment
 that is essential to determining trustworthiness.  Generally speaking, each claim
-has a name (or other ID)
-and a singleton value, being the value of that specific Attester at a given point
+has a name (or key, label, or other ID)
```
Comment (thomas-fossati) [inline draft-ietf-rats-endorsements.md]: There is no need to add _"key, label,"_ as they are already subsumed by "other ID".

File: draft-ietf-rats-endorsements.md [hunk 5]
```diff
@@ -62,73 +62,73 @@ message format for Endorsements in the scope of the RATS architecture.
...
 in time. Some claims may inherently have multiple values, such as a list of
-files in a given location on the device, but for our purposes we will treat such
-a list as a single unit, meaning one Attester at one point in time.
+files in a given location on the device, but in the context of this document such
+a list is treated as a single unit, representing one Attester at one point in time.
 
-"Reference state" is a group of claimsets about the desired or undesired state of
-the Attester.  Typically, each claim has a name (or other ID) and
+"Reference state" is a group of claim sets about the desired or undesired state of
+an Attester.  Typically, each claim has a name (or key, label, or other ID) and
```
Comment (thomas-fossati) [inline draft-ietf-rats-endorsements.md]: ditto

File: draft-ietf-rats-endorsements.md (hunk 1 above)
Comment (henkbirkholz) [inline draft-ietf-rats-endorsements.md]:
```suggestion
manner, i.e., a tree.
The state of an Attester also encompasses the combination of static and
```

used former. removed generic part in the redundancy, for the sake simplicity

File: draft-ietf-rats-endorsements.md [hunk 6]
```diff
@@ -62,73 +62,74 @@ message format for Endorsements in the scope of the RATS architecture.
...
-messages with information about actual state, and information about desired/undesired
+conceptual messages with information about actual state, and information about desired/undesired
 states, and an appraisal policy that controls how the two are compared.
 
 Each Attester in general has at least one Attesting Environment and one Target
 Environment (e.g., hardware, firmware, Operating System, etc.).  Typically, each
 Attester has multiple Target Environments, each with their own set of claims (sometimes
-called a "claimset") representing their actual state.  Additionally, the number of
-Target Environments is not limited.
+called a "claim sets") representing their actual-state.  Additionally, the number of
```
Comment (henkbirkholz) [inline draft-ietf-rats-endorsements.md]:
```suggestion
called a "claim sets") representing their actual state.  Additionally, the number of
```

File: draft-ietf-rats-endorsements.md (hunk 4 above)
Comment (henkbirkholz) [reply draft-ietf-rats-endorsements.md]:
```suggestion
has a name (typically referred to as label and occasionally referred to as a key, code-point, or some other ID)
```

Comment (thomas-fossati) [reply draft-ietf-rats-endorsements.md]: that is making things worse, but it's ok :-)

File: draft-ietf-rats-endorsements.md [hunk 7]
```diff
@@ -62,73 +62,74 @@ message format for Endorsements in the scope of the RATS architecture.
...
 in time. Some claims may inherently have multiple values, such as a list of
-files in a given location on the device, but for our purposes we will treat such
-a list as a single unit, meaning one Attester at one point in time.
+files in a given location on the device, but in the context of this document such
+a list is treated as a single unit, representing one Attester at one point in time.
 
-"Reference state" is a group of claimsets about the desired or undesired state of
-the Attester.  Typically, each claim has a name (or other ID) and
+"Reference state" is a group of claim sets about the desired or undesired state of
+an Attester.  Typically, each claim has a name (or key, label, or other ID) and
```
Comment (henkbirkholz) [inline draft-ietf-rats-endorsements.md]:
```suggestion
an Attester.  Typically, each claim has a name and
```

**Review by thomas-fossati — APPROVED** [review]

**Pull request merged**